app/locales/en/translations.js

```diff
@@ -0,0 +1,233 @@
+export default {
+    global: {
+        share: `Share`,
+        cancel: `Cancel`,
+        prev: `Prev`,
+        next: `Next`,
+        none: `None`,
+        abstract: `Abstract`,
+        doi: `DOI`,
+        tags: `Tags`,
+        search: `Search`,
+        brand: `OSF Preprints`,
+        add_preprint: `Add a preprint`,
+        title: `Title`,
+        search_preprints: `Search preprints...`,
+        added_on: `Added on`,
+        add: `Add`,
+        restart: `Restart`,
+        no_results_found: `No results found.`,
+        authors: `Authors`
+    },
+    _404: {
```

Comment:
I could live with this either way but just to make a note you can also quote a key to handle numbers, spaces and special characters.